### PR TITLE
posted screenshots in README.md

### PR DESCRIPTION
**Pull request by owner6**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,3 +68,11 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
...
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+
+# Screenshots
+
+[1st person position recognition](https://github.com/owner6/tfjs-diploma/blob/master/screenshots/photo_2022-09-06_02-51-03.jpg)
+[recognition of the position of several people at once](https://github.com/owner6/tfjs-diploma/blob/master/screenshots/photo_2022-09-06_02-51-03%20(2).jpg)
+[architecture resNet50](https://github.com/owner6/tfjs-diploma/blob/master/screenshots/photo_2022-09-06_02-56-46.jpg)
+[architecture mobileNetV1](https://github.com/owner6/tfjs-diploma/blob/master/screenshots/photo_2022-09-06_02-57-10.jpg)
+[the code](https://github.com/owner6/tfjs-diploma/blob/master/screenshots/photo_2022-09-06_02-59-25.jpg)
```
Comment (dever4eg) [inline README.md]: Why did you add this screenshot? It is part of code..

File: README.md [hunk 2]
```diff
@@ -68,3 +68,11 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
 ### `npm run build` fails to minify
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+
+# Screenshots
+
+[1st person position recognition](https://github.com/owner6/tfjs-diploma/blob/master/screenshots/photo_2022-09-06_02-51-03.jpg)
+[recognition of the position of several people at once](https://github.com/owner6/tfjs-diploma/blob/master/screenshots/photo_2022-09-06_02-51-03%20(2).jpg)
+[architecture resNet50](https://github.com/owner6/tfjs-diploma/blob/master/screenshots/photo_2022-09-06_02-56-46.jpg)
+[architecture mobileNetV1](https://github.com/owner6/tfjs-diploma/blob/master/screenshots/photo_2022-09-06_02-57-10.jpg)
```
Comment (dever4eg) [inline README.md]: The interface is already included in the main screenshot. I think you can remove these ones

File: README.md [hunk 3]
```diff
@@ -68,3 +68,11 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
 ### `npm run build` fails to minify
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+
+# Screenshots
```
Comment (dever4eg) [inline README.md:84]: It is not shown. Have you tested?

image

Comment (dever4eg) [inline README.md:84]: Also I think it is better to move screenshots to the top of readme, because it is the most interesting part

**Comment by dever4eg** [comment]
@owner6 Why did you update package-lock ? It should not be in this PR

**Comment by dever4eg** [comment]
@owner6 You forgot to create a branch for this feature.

File: README.md (hunk 1 above)
Comment (owner6) [reply README.md]: > Why did you add this screenshot? It is part of code..

thanks, deleted

File: README.md (hunk 2 above)
Comment (owner6) [reply README.md]: > 

I suggest you consider the option that the screenshots below are displayed in this file. Did I do the right thing by creating a folder with screenshots in the root of the project?

Comment (owner6) [reply README.md]: please check readme.md. for correctness
